rfc6962-bis.xml

```diff
@@ -1101,13 +1101,7 @@ messages.
           </list>
         </t>
         <t>
-          Note that this message is not signed -- the <spanx style="verb">entries</spanx> data can be
-verified by constructing the Merkle Tree Hash corresponding to a retrieved
-STH. All leaves MUST be v1 or v2. However, a compliant v1 client MUST NOT construe an
-unrecognized LogEntryType value as an error. This means it
-may be unable to parse some entries, but note that each client can inspect the
-entries it does recognize as well as verify the integrity of the data by
-treating unrecognized leaves as opaque input to the tree.
+          Note that this message is not signed -- the <spanx style="verb">entries</spanx> data can be verified by constructing the Merkle Tree Hash corresponding to a retrieved STH. All leaves MUST be v2. However, a compliant v2 client MUST NOT construe an unrecognized TransItem value as an error. This means it may be unable to parse some entries, but note that each client can inspect the entries it does recognize as well as verify the integrity of the data by treating unrecognized leaves as opaque input to the tree.
```

Comment:
"unrecognized TransItem value".  It's specifically the type of the TransItem that may be unrecognized.

I suggest: s/TransItem value/TransItem type/